sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs

```diff
@@ -115,11 +122,103 @@ public IScaleMonitor GetMonitor()
         {
             return _scaleMonitor.Value;
         }
-
+        
         public ITargetScaler GetTargetScaler()
         {
             return _targetScaler.Value;
         }
+        
+        internal class EventProcessorStoredEventsManager : IDisposable
+        {
+            private SemaphoreSlim _storedEventsGuard;
+            private int _maxBatchSize;
+            private int _minBatchSize;
+
+            // This is internal for mocking purposes only.
+            internal ConcurrentDictionary<string, List<EventData>> StoredEvents { get; set; }
```

Comment:
Given that we only have one set of events and no longer have to worry about partitions... do we need the dictionary at all or can we JUST use a `ConcurrentQueue<EventData>`?